fabtests/include/windows/osd.h

```diff
@@ -46,6 +46,7 @@ struct iovec
 #define strdup _strdup
 #define strcasecmp _stricmp
 #define strncasecmp _strnicmp
+#define alloca _alloca
```

Comment:
We should look at creating symbolic links from libfabric/fabtests/include/windows to libfabric/include/windows, so we only have a single copy of these portability files.  It's probably not quite that easy, as we'd also need the implementation source files linked in some cases.